features/promotion/reapplying_promotion_on_cart_change.feature

```diff
@@ -33,28 +33,28 @@ Feature: Reapplying promotion on cart change
         And my cart shipping fee should be "€30.00"
         And my discount should be "-€30.00"
 
-    @todo
+    @ui
     Scenario: Receiving discount after removing an item from the cart and then adding another one
         Given the store has a product "Symfony T-Shirt" priced at "€150.00"
         And the promotion gives "€10.00" fixed discount to every order
         And I had product "PHP T-Shirt" in the cart
-        But I removed this product from the cart
+        But I removed product "PHP T-Shirt" from the cart
```

Comment:
This is a temporary change. After merging https://github.com/Sylius/Sylius/pull/4269 we could use ``this product`` or ``it`` without injecting ``SharedStorage`` to ``CartContext``.